src/styles/blocks/catalog-block.scss

```diff
@@ -0,0 +1,107 @@
+.catalog {
+  width: $catalog-width;
+  display: flex;
+  flex-wrap: wrap;
+  padding: $catalog-vertucal-padding $catalog-horizontal-padding;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  margin-right: $card-margin-right;
+  background: #fff;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__text {
+    font-weight: $font-weight-medium;
+    font-size: $font-size-m;
```

Comment:
add this styles for the whole page and just change them where you need it

Reply:
I understood correctly, that everywhere (in the entire file) where these values occur, they must be replaced with these variables, right?